sencha-workspace/packages/slate-cbl/src/view/student/skill/OverviewWindowController.js

```diff
@@ -79,13 +75,16 @@ Ext.define('Slate.cbl.view.student.skill.OverviewWindowController', {
         var me = this,
             demonstrationsTable = me.lookupReference('demonstrationsTable'),
             skillId = me.lookupReference('skillCombo').getValue(),
-            studentId = me.view.getStudent();
-
+            studentId = me.view.getStudent(),
+            demonstrationId = me.view.getDemonstration();
+            
         if (skillId && studentId) {
-            demonstrationsTable.setLoading('Loading demonstrations&hellip;'); // currently not visible due to http://www.sencha.com/forum/showthread.php?290453-5.0.x-loadmask-on-component-inside-window-not-visible
+            // currently not visible due to http://www.sencha.com/forum/showthread.php?290453-5.0.x-loadmask-on-component-inside-window-not-visible
+            demonstrationsTable.setLoading('Loading demonstrations&hellip;');
             Slate.cbl.model.Skill.load(skillId, {
                 callback: function(skill) {
                     skill.getDemonstrationsByStudent(studentId, function(skillDemonstrations) {
+                        demonstrationsTable.selectedDemonstrationId = demonstrationId;
```

Comment:
Why are we stashing this value in `demonstrationsTable.selectedDemonstrationId`? I don't see it getting used anywhere in this PR